CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Version 282:
+
+* file_win32 supports UTF-8 paths
+* file_stdio supports unicode paths
```

Comment:
More correct: `file_stdio supports UTF-8 paths on MSVC` :/

Reply:
I guess that is more accurate..are you saying that other toolchains on windows don't use the dinkum stdlib or whatever it is that msvc uses? I am ignorant about such things (I pretty much stay in visual studio always).

Reply:
No, it really is the MSVC compiler only due to usage of `BOOST_MSVC`: 

> Defined if the compiler is really Microsoft Visual C++, as opposed to one of the many other compilers that also define _MSC_VER. Has the same value as _MSC_VER. 

from https://www.boost.org/doc/libs/master/libs/config/doc/html/boost_config/boost_macro_reference.html

In #1791 I proposed to use `_wfopen` if on windows but not on MSVC and adjust the checks accordingly. Then it will either fail to compile (unlikely as AFAIK all windows compiler libs have `_wfopen`) or work as advertised which is a desirable outcome